<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoggerWnd</class>
 <widget class="QWidget" name="LoggerWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Logger</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QMainWindow {
	background-color: #313131;
}

QTextEdit {
	color: white;
    border: 1px solid #505050;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #181818, stop: 1.0 #000000);
	font-family: Courier, &quot;Courier New&quot;, monospace;
}

QTextEdit:focus
{
    border: 1px solid #505050;
	background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #181818, stop: 1.0 #000000);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtLog">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>22</y>
      <width>600</width>
      <height>478</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSaveAs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Save As ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPerf">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Performance</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbLevel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
